examples/ssr/package.json

```diff
@@ -16,6 +19,13 @@
     "react-dom": "17.0.2"
   },
   "devDependencies": {
-    "razzle": "^3.4.5"
+    "babel-preset-razzle": "^4.2.14",
+    "html-webpack-plugin": "^5.2.0",
+    "mini-css-extract-plugin": "0.9.0",
+    "postcss": "8.2.4",
```

Comment:
Why is this a fixed version?

Reply:
as requested by Razzle 
https://razzlejs.org/docs/upgrade-guide

```
yarn add --dev \
  webpack-dev-server@3.11.0 \
  mini-css-extract-plugin@0.9.0 \
  postcss@8.2.4
```